src/main/java/hattrickdata/CurrentCapacity.java

```diff
@@ -0,0 +1,26 @@
+package hattrickdata;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+@Getter
+@SuperBuilder()
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class CurrentCapacity extends Capacity {
+
+    private LocalDateTime rebuildDate;
```

Comment:
I would be happy, if we could change this to HODateTime.  

HODateTime was the replacement of a lot of different DateTimes in HO code. Please do not introduce those again.

Reply:
Yes, I saw it right after PR creation too.

Reply:
Refactor: Review Finding 5: Use HODateTime instead of LocalDateTime